torch/csrc/jit/passes/erase_number_types.cpp

```diff
@@ -16,7 +16,7 @@ static void EraseNumberTypesOnBlock(Block* block) {
         if(it->output()->type()->isSubtypeOf(NumberType::get())) {
           auto s = *constant_as<at::Scalar>(it->output());
           WithInsertPoint guard(*it);
-          Value* r = block->owningGraph()->insertConstant(s.toTensor());
+          Value* r = block->owningGraph()->insertConstantWithScope(s.toTensor(), it->scopeName());
```

Comment:
I really don't think we need `insertConstantWithScope`, since this line can basically be
```cpp
Value * r = block->owningGraph()->insertConstantWith(s.toTensor());
r->node()->setScope(it->scope());
```